cypress/projects/vite/FrameComponent.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Can we convert this, and other "entrypoint" files (e.g. components and snippets) to TypeScript? That way we have an easy way to test out vite + TypeScript support.

Reply:
yeah good idea, changing!